src/components/Dashboard/TodoList.vue

```diff
@@ -403,7 +403,7 @@
                   </v-btn>
                   <v-btn
                     v-else
-                    class="btn-resume-payment"
+                    :class="{ 'btn-resume-payment': !isCancellableTodoItem(item) }"
```

Comment:
I wasn't going to pick on this, but I think I will anyway (sorry).

Instead of removing the classname if this item isn't cancellable, why not add a classname if it isn't cancellable, and adjust the styles below accordingly? (as in, "adjust the border if the selector does not include cancellable")

That way, this button always has a classname.

Reply:
Ref: https://developer.mozilla.org/en-US/docs/Web/CSS/:not

Reply:
No worries!

Sev, I'm a bit confused here though. My thought was that having no class looks good since we don't want to set the borders to 0 if it's not cancellable. So basically: 

If not cancellable => no setting of the border (no class).
If cancellable => set border to 0 (the class is there).

So having a class when it isn't cancellable will not have any properties since I believe it looks good as is.

Please let me know if I'm missing something so that I can fix it!

Reply:
Hey man, you always want to be classy 😉 

It's a button. It should have some way to identify or select it in the DOM (eg, for testing). Even if the class isn't used to add CSS properties.

Reply:
Well, I am your student 😎 

OK, got it. I'll have a class for both cases then Sev!

Reply:
Just a second, dynamic class, in addition to the existing classname, right?

Reply:
Sev, sorry for taking too long to reply. My head, for some reason, wasn't processing this although it was super simple haha.

Please have a look at last commit, hopefully it's fine now!